src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+//
+// The Radworks Timelock is not compatible with the Governor Alpha Timelock interface. This
+// meant we had to fork the contracts that interact with the timelock and replace the Timelock
+// interface with the Radworks Timelock interface. Also, if Radworks decides to change the
+// Timelock in the future then it must conform to the Radworks Timelock interface.
+//
+// Radworks Timelock interface:
+// https://etherscan.io/address/0xB3a87172F555ae2a2AB79Be60B336D2F7D0187f0#code#F1#L306
+// Governor Alpha Timelock interface:
+// https://github.com/compound-finance/compound-protocol/blob/a3214f67b73310d547e00fc578e8355911c9d376/contracts/Governance/GovernorAlpha.sol#L320
+// forgefmt: disable-start
```

Comment:
great idea here; when i do a diff between the two files though, the whole thing comes up as modified. perhaps `forge fmt` was run after all?

Reply:
Yes.. forge fmt was run... also at line 10, that is not the right etherscan URL (copy pasta issue) now updated.
4c8b4aa5079ff2ca8a0124eeab3624cc5cfce0db

Reply:
hmm, i'm confused why `forge fmt` formatted this file when line 13 disables forge fmt. I do think it makes sense to disable forge fmt on external files where the diff needs to be clean. but I don't think it should block merge here.